Game/world/world.h

```diff
@@ -175,6 +176,19 @@ class World final {
     void                 addFreePoint  (const Tempest::Vec3& pos, const Tempest::Vec3& dir, const char* name);
     void                 addSound      (const ZenLoad::zCVobData& vob);
 
+    inline bool addItemOrNpcBySymbolName (const std::string& name, const Tempest::Vec3& at) {
```

Comment:
That is not what I meant by saying "inline" :)
My point is: there is not reason for this method to exist in `World` class - this code can be just moved into `marvin.cpp` under C_Insert case

Reply:
Ahhhhh, that makes muuuuuuuch more sense.
OK will do that, sorry for the misunderstanding.